src/java.management/share/classes/java/lang/management/MemoryMXBean.java

```diff
@@ -267,6 +267,43 @@ public interface MemoryMXBean extends PlatformManagedObject {
      */
     public MemoryUsage getNonHeapMemoryUsage();
 
+    /**
+     * Returns the approximate accumulated time, in nanoseconds,
+     * spent in garbage collection.
+     *
+     * <p> This is the CPU time used by all garbage collection
+     * activity, including any overhead, which means the result
+     * may be non-zero even if no GC has occurred.
+     *
+     * This method returns {@code -1} if the platform does
+     * not support this operation or the information is not
+     * available.
+     *
+     * @apiNote
+     * May be used in conjunction with {@link jdk.management/com.sun.management.OperatingSystemMXBean#getProcessCpuTime()}
+     * for calculating the GC's usage of CPU time as a whole.
+     *
+     * @implNote  The specifics on what constitutes the time spent
+     * in garbage collection is highly implementation dependent.
+     * In the HotSpot Virtual Machine implementation reported
+     * time will include relevant implementation-specific details such
```

Comment:
"In the HotSpot Virtual Machine implementation reported time will include".  A suggestion to improve this is to change it to "In the HotSpot Virtual Machine, this time includes.."